third_party/serf/UPDATING_BORINGSSL

```diff
@@ -26,6 +40,3 @@ http://googlechromereleases.blogspot.com/search/label/Stable%20updates
     cp -r src/third_party/boringssl/$arch $SERF_DIR/$arch/
   done
 
```

Comment:
Dunno if it's worth calling out that you need to cd back into the git directory, commit stuff and push? It's probably also a good idea to spin a build before pushing at least.

But this looks much better, thanks.